wwwapp/management/commands/download_remote_images.py

```diff
@@ -40,13 +40,19 @@ def handle(self, *args, **options):
     def update_description(text, path, dryrun):
         def update_img(m):
             url = m.group(1)
+            if not url.startswith("http://") and not url.startswith("https://"):
+                return m.group(0)
             print("Downloading {}".format(url))
             r = requests.get(url)
             if r.status_code != 200:
                 print("WARNING: {} returned {}".format(url, r.status_code))
                 return m.group(0)
             else:
-                ext = mimetypes.guess_extension(r.headers['Content-Type'])
+                if 'Content-Type' not in r.headers:
+                    raise Exception("Server didn't provide a Content-Type for the file")
```

Comment:
You shouldn't trust the mime type returned from the server but rather run a manual check with for instance https://pypi.org/project/filetype/

Reply:
This is just for the extension, I could just take the end from the filename and achieve the same result (but it would break on pages that do weird magic like serving images from scripts). And I'm not introducing permanent dependencies for a quick one-off thing. The dry run option is there for a reason, you are supposed to verify that it's not going to do anything funny manually.